code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -308,6 +308,11 @@
 	item_state = "tile-bcircuit"
 	turf_type = /turf/open/floor/circuit
 	custom_materials = list(/datum/material/iron = 500, /datum/material/glass = 500) // WS Edit - Item Materials
+	tile_reskin_types = list(
+		/obj/item/stack/tile/circuit,
+		/obj/item/stack/tile/circuit/green,
+		/obj/item/stack/tile/circuit/red
```

Comment:
```suggestion
		/obj/item/stack/tile/circuit/red,
```

Reply:
isnt this literally what broke tile selection the last time

Reply:
opposite, having the comma at the end of the list broke it, not not having the comma

Reply:
yeah thats what i mean, youre removing the commas to fix it

Reply:
If having the list done correctly is causing errors, I'll checkout the branch locally to figure out why because that's the correct way of doing long lists